x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/lucene/BlockDocValuesReader.java

```diff
@@ -78,6 +78,7 @@ public static BlockDocValuesReader createBlockReader(
             final SortedBinaryDocValues bytesValues = bytesVS.bytesValues(leafReaderContext);
             return new BytesValuesReader(bytesValues);
         }
+        // TODO: Add support for geo_point and geo_shape
```

Comment:
This feels like an important part of testing this PR.

Reply:
At least, the geo_point part.

Reply:
Wait. You have an example that loads a geo_point field from ES. How do?

Reply:
Turns out that the CsvTests don't touch this code path, but the EsqlSpecIT do, so those are failing in the PR. I started working on this last week, but did not yet push a fix.